source/jormungandr/jormungandr/scenarios/experimental.py

```diff
@@ -575,192 +42,124 @@ def __init__(self):
+                                                               request=request,
+                                                               streetnetwork_path_pool=None)
+
+        orig_places_free_access = PlacesFreeAccess(instance=instance, requested_place_obj=requested_orig_obj)
+        dest_places_free_access = PlacesFreeAccess(instance=instance, requested_place_obj=requested_dest_obj)
+
+        orig_fallback_durations_pool = FallbackDurationsPool(instance=instance,
+                                                             requested_place_obj=requested_orig_obj,
+                                                             modes=requested_dep_modes,
+                                                             proximities_by_crowfly_pool=orig_proximities_by_crowfly,
+                                                             places_free_access=orig_places_free_access,
+                                                             streetnetwork_path_pool=streetnetwork_path_pool,
+                                                             request=request)
+
+        dest_fallback_durations_pool = FallbackDurationsPool(instance=instance,
+                                                             requested_place_obj=requested_dest_obj,
+                                                             modes=requested_arr_modes,
+                                                             proximities_by_crowfly_pool=dest_proximities_by_crowfly,
+                                                             places_free_access=dest_places_free_access,
+                                                             streetnetwork_path_pool=None,
+                                                             request=request)
+
+        pt_journey_pool = PtJourneyPool(instance=instance,
+                                        requested_orig_obj=requested_orig_obj,
+                                        requested_dest_obj=requested_dest_obj,
+                                        streetnetwork_path_pool=streetnetwork_path_pool,
+                                        krakens_call=krakens_call,
+                                        orig_fallback_durations_pool=orig_fallback_durations_pool,
+                                        dest_fallback_durations_pool=dest_fallback_durations_pool,
+                                        request=request)
+
+        completed_pt_journeys = wait_and_complete_pt_journey(requested_orig_obj=requested_orig_obj,
+                                                             requested_dest_obj=requested_dest_obj,
+                                                             pt_journey_pool=pt_journey_pool,
+                                                             streetnetwork_path_pool=streetnetwork_path_pool,
+                                                             orig_places_free_access=orig_places_free_access,
```

Comment:
I thought of that too, but no eventually. `places_free_access` has two types of places: `stop_points` attached to a `stop_area` and `odt_stop_points` of a `place`, according to the type, the pt_journeys are built differently. you can read [this](https://github.com/CanalTP/navitia/pull/2046/files/f006ba90dc541423e1b5e1ec11985116ef1971d2#diff-9ca4efa8d09d46bc612e4aa3588368b4R138) to see the different treatment. I can merge it into `fallback_durations` by adding a type into the dictionary but i'm not convinced it'll be more clear to read